.github/scripts/comment-pr.js

```diff
@@ -0,0 +1,69 @@
+module.exports = async ({ github, context }) => {
```

Comment:
Shall we change the file name a bit more specific so that this script is for netlify?

Reply:
How would you name this file?